Essentials/src/plugin.yml

```diff
@@ -527,3 +527,1183 @@ permissions:
+    description: This allows you to use the matrix/magic color in your private messages.
+    default: false
+  essentials.msg.multiple:
+    description: This allows you to message multiple users in a single command.
+    default: false
+  essentials.msg.url:
+    description: This allows you to use urls in your private messages.
+    default: false
+  essentials.near:
+    description: Allow access to the /near command.
+    default: false
+  essentials.near.maxexempt:
+    description: Allows bypassing the radius limit.
+    default: false
+  essentials.near.others:
+    description: Allows using the near command on another player.
+    default: false
+  essentials.nick:
+    description: Allow access to the /nick command.
+    default: false
+  essentials.nick.color:
+    description: This allows you to color your nickname using color codes.
+    default: false
+  essentials.nick.format:
+    description: This allows you to format your nickname using format codes.
+    default: false
+  essentials.nick.magic:
+    description: This allows you to use the matrix/magic color in your nickname.
+    default: false
+  essentials.nick.others:
+    description: Gives you permission to give other players a nick name.
+    default: false
+  essentials.powertool:
+    description: Allow access to the /powertool command.
+    default: false
+  essentials.powertool.append:
+    description: Allows adding multiple commands to a single powertool.
+    default: false
+  essentials.powertooltoggle:
+    description: Allow access to the /powertooltoggle command.
+    default: false
+  essentials.msg:
+    description: Allow access to the /msg command.
+    default: false
+  essentials.msg.color:
+    description: This allows you to color your private messages using color codes.
+    default: false
+  essentials.msg.format:
+    description: This allows you to format your private messages using format codes.
+    default: false
+  essentials.msg.magic:
+    description: This allows you to use the matrix/magic color in your private messages.
+    default: false
+  essentials.msg.url:
+    description: This allows you to use urls in your private messages.
+    default: false
+  essentials.realname:
+    description: Allow access to the /realname command.
+    default: false
+  essentials.recipe:
+    description: Allow access to the /recipe command.
+    default: false
+  essentials.rules:
+    description: Allow access to the /rules command.
+    default: false
+  essentials.seen:
+    description: Allow access to the /seen command.
+    default: false
+  essentials.seen.banreason:
+    description: Allows the user to see why a user is banned.
+    default: false
+  essentials.seen.extra:
+    description: Allow the user to see the last IP address a user connected with, and their last logged in location.
+    default: false
+  essentials.seen.ipsearch:
+    description: Allows the user to search IPs for username matches.
+    default: false
```

Comment:
The `essentials.spawner.[mob]` and `essentials.spawner.*` perms are presumably only for `/spawner` and not `/spawnmob`.